pages/code-review.md

```diff
@@ -14,28 +14,61 @@ channel!
 
 ## Why reviews?
 
-Code reviews are incredibly important step to the development process. Our goals
-when reviewing code are to:
+Code reviews are incredibly important step to the development process. Our
+goals when reviewing code are to:
 
 - expose bugs before they make it to production;
 - ensure a consistent quality in the codebase;
 - create an environment for sharing knowledge and developing skills;
+- cross-pollination of debugging skills when problems arise;
+- cultivation of the ability to critique one’s own code more strongly;
```

Comment:
To make the bullet points more consistent grammatically, they should start with verbs, like the first three points. So, "cross-pollinate debugging skills..." and "cultivate the ability...". Also, I think "one's own" is redundant. "one's code" reads better to me.